hazelcast/src/test/java/com/hazelcast/datalink/HazelcastDataLinkTest.java

```diff
@@ -117,6 +121,28 @@ public void shared_client_should_return_same_instance() {
         }
     }
 
+    @Test
```

Comment:
What do you think about tests with:
- non-existing config file
- invalid config file
- mixed properties (client_xml/client_xml_path) set
- mixed properties (client_xml/client_yml_path) set